### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by Haxarus** [Soreepeong/XivAlexander#313]
![image](https://user-images.githubusercontent.com/41932290/149087438-ea8ecadb-0573-4421-8537-90afc77794e2.png)

Logs not working

**Comment by YuleXiv** [comment]
Are you, by any chance, using Gshade?  did you install Gshade first, and then Alexander?

**Comment by Haxarus** [comment]
Yeah I did do that, however I uninstalled gshade, didnt like it. Then reinstalled alexander, still didnt work. (I deleted all of the files).
Then I also reinstalled the game, then reinstalled alexendar, still didnt work. Other than the stuff in the game folder, is there another location with files? Maybe the appdata folder?

**Comment by Soreepeong** [comment]
"Not working" => What's not working?

"Logs not working" => How is logs not working?

Screencap => Unloading feature is provided as "if possible". It may not be possible at all at times.

**Comment by Haxarus** [comment]
What's not working?/How is logs not working?
The first couple connection lines appear in the logs, after that nothing. So when I hit a button, nothing appears, hence Alexander not working.

I have unloaded several times in several different cases.

I should mention this though, if I use the option "load into game once" it works. However once I close the game, the game freezes and I need to force shutdown the game, after that I get a different error from Alexander.

**Comment by Soreepeong** [comment]
Exit related problems should be fixed in the next release.

For no logs being recorded when you use actions, first, check if you do have `Use Logging` for `Networking > Latency and Timing Helper` ticked.

Do you use a VPN? In that case, try ticking everything in `Networking > Troubleshooting`.

**Comment by Haxarus** [comment]
I do use a VPN, however I did try without VPN as well. I also tried Networking>Troubleshooting tip, lines do appear for a while.

For the Networking>Latency and timing helper tip, its ticked.  I would be able to feel it in the gameplay if Alexander was working anyway.

**Comment by Soreepeong** [comment]
Now that stuff show up in log after ticking stuff in troubleshooting, save the logs from the log window and post it.

**Comment by Haxarus** [comment]
[XivAlexander_20220117_174718_057.log](https://github.com/Soreepeong/XivAlexander/files/7879612/XivAlexander_20220117_174718_057.log)

Thanks for your help btw.

**Comment by Soreepeong** [comment]
You will have to extract everything in the zip then Install or Load Once, or the program will not work. Try again after downloading and extracting everything.

**Comment by Haxarus** [comment]
Thats what I tried, it was a fresh copy of the program, just downloaded before.

**Comment by Soreepeong** [comment]
Oh, I think I get what's going on. Open `G:\Steam Games\steamapps\common\FINAL FANTASY XIV Online\game\`, delete `config.init.json`, and then try again. It should definitely work when you do that and restart the game.

**Comment by Haxarus** [comment]
Thanks a lot!
Deleting the .json file didnt work, however pasting the opcode worked.

**Comment by Haxarus** [comment]
Also looking forward to the update to address the close issue. Feel free to close issue.

**Comment by Soreepeong** [comment]
The problem is, how did you get into the situation you were in to begin with? Can you take a screencap of `%APPDATA%\XivAlexander` and `%LOCALAPPDATA%\XivAlexander`? Just type `Windows+R`, and paste one, and press enter.

**Comment by Haxarus** [comment]
![image](https://user-images.githubusercontent.com/41932290/149727914-400550a8-e0f7-412f-8f83-afaf3d038df1.png)

![image](https://user-images.githubusercontent.com/41932290/149727959-e7dfc094-3292-4ca9-b867-078a3d7a2621.png)



Couple of weeks ago, I played around with Gshade, it messed up something with the game so I uninstalled. However I didn't know it was Gshade at the start. I uninstalled and reinstalled both gshade and Alexander multiple times. I think I broke something then.
